dlio_benchmark/checkpointing/base_checkpointing.py

```diff
@@ -431,7 +431,7 @@ def load_checkpoint(self, epoch, step_number):
         self.checkpoint_storage.create_node(checkpoint_id, exist_ok=True)
         if self.rank_to_checkpoint == my_rank:
             if self.model_state:
-                self.load_state(suffix=f"{checkpoint_id}/model_states-{my_rank}", state=self.model_state, fsync = self.args.checkpoint_fsync)
```

Comment:
Why was fsync removed? revert this.

Reply:
by default in `base_checkpointing.py`, `load_state` method does not have `fsync` as it is parameter. this resulting in undetected error (our test-cases previously did not catch this)

Reply:
Ohk.